optimum/habana/transformers/models/gemma2/modeling_gemma2.py

```diff
@@ -0,0 +1,1005 @@
+# coding=utf-8
+# Copyright 2023 Mistral AI and the HuggingFace Inc. team. All rights reserved.
```

Comment:
incorrect license. shouldnt refer to gpt-neox, should refer to Google.

https://github.com/huggingface/transformers/blob/main/src/transformers/models/gemma2/modeling_gemma2.py

Reply:
Fixed. Thanks!